test/dynamo/test_exc.py

```diff
@@ -37,7 +37,13 @@ def fn001(x):
                 torch.randn(1)
             ),
             """\
-'skip function graph_break in file _dynamo/decorators.py'
+Attempted to call function marked as skipped
+  Explanation: Dynamo developers have manually marked that the function `graph_break`\
+ in file _dynamo/decorators.py should not be traced.
+  Hint: Avoid calling the function `graph_break`.
+
+  Developer debug context: module: torch._dynamo.decorators, qualname: graph_break, skip reason: <missing reason>
```

Comment:
torch._dynamo.graph_break should get its own error message.

"GraphBreak due to a call to torch._dynamo.graph_break"?

Maybe we should let the user provide an error message?